js/north.js

```diff
@@ -44,6 +44,33 @@
 			} );
 		};
 
+		$.fn.smoothScroll = function () {
```

Comment:
Because we're attaching this to the global jQuery object, I'd suggest we namespace this in some way. Maybe going with northSmoothScroll.